tests/CI/docker-compose.yml

```diff
@@ -121,6 +121,8 @@ services:
     user: "${DIRAC_UID}:${DIRAC_GID}"
     depends_on:
       - dirac-server
+    volumes:
+      - /cvmfs:/cvmfs:ro
```

Comment:
Does this mean people can no longer run the integration tests if they don't have cvmfs mounted?

Reply:
Good point. I added a workaround.